content/shared/influxdb3-processing-engine.md

```diff
@@ -0,0 +1,84 @@
+# Processing engine
+
+The Processing engine is an embedded Python virtual machine that runs inside an {{% product-name %}} database server. It executes Python code in response to triggers and database events without requiring external application servers or middleware.
+
+## How it works
+
+### Architecture
+
+The Processing engine runs Python code directly within a {{% product-name %}} server process. This design provides high performance and direct access to database resources.
+
+- **Embedded execution**: Code runs in the same process space as the database server
+- **Direct data access**: Zero-copy access to data
+- **Event-driven**: Responds to database writes, scheduled events, and HTTP requests
+- **Isolated contexts**: Maintains separation between different plugin executions
+- **Cache integration**: Access to system caches including Last values  and Distinct values 
```

Comment:
```suggestion
- **Cache integration**: Access to system caches, such as Last Value Cache  and Distinct Value Cache 
```

We'll be publishing cache docs for GA tomorrow. You can link the LVC and DVC terms to those.